pkg/kubectl/cmd/version.go

```diff
@@ -37,6 +37,14 @@ type Version struct {
 	ServerVersion *apimachineryversion.Info `json:"serverVersion,omitempty" yaml:"serverVersion,omitempty"`
 }
 
+// VersionConfig: describe the options available to users of the "kubectl
+// version" command.
+type VersionConfig struct {
```

Comment:
I had originally called it `VersionOptions` but I changed it to match the convention stated in [kubectl-conventions.md](https://github.com/kubernetes/community/blob/49d65710b3ef8f4326df79706daf1e59c2e4a465/contributors/devel/kubectl-conventions.md#command-implementation-conventions). Thoughts?